### ISSUE TITLE
Improve color contrast in the footer

### ISSUE THREAD

**Issue by mgifford** [trbouma/mainstay#2]
Title: WCAG 1.4.3: Ensure the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds (.md-copyright)
Tags: Accessibility, WCAG 1.4.3, color-contrast

Issue: Ensure the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds (color-contrast - https://accessibilityinsights.io/info-examples/web/color-contrast)

Target application: Mainstay - Mainstay - https://mainstay.safebox.dev/

Element path: .md-copyright

Snippet: <div class="md-copyright">

Related paths: 
  .md-footer-meta
  footer
  body

How to fix: 
Fix any of the following:
  Element has insufficient color contrast of 4.47 (foreground color: #7f7f7f, background color: #171717, font size: 9.6pt (12.8px), font weight: normal). Expected contrast ratio of 4.5:1

Environment: Chrome version 151.0.0.0

====

This accessibility issue was found using Accessibility Insights for Web 2.47.0 (axe-core 4.11.3), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.
